src/features/user/TreeMapper/Analytics/components/ProjectFilter/index.tsx

```diff
@@ -96,8 +100,9 @@ const ProjectFilter = () => {
           >
             <MuiDatePicker
               label={t('to')}
-              value={toDate}
-              onChange={setToDate}
+              value={localToDate}
+              onChange={(e) => setLocalToDate(e)}
+              onAccept={setToDate}
```

Comment:
Resolve type warnings due to mismatch in `onAccept` type and `setToDate`/`setFromDate`

Reply:
Resolved in 6caa3eee61093a4c0219e95609aa3ed9765d85e7